Plugin~/Src/MeshSync/Include/MeshSync/SceneGraph/msCamera.h

```diff
@@ -11,6 +11,7 @@ using super = Transform;
     // serializable
     CameraDataFlags cd_flags;
     bool is_ortho;
+    float ortho_size;
```

Comment:
@sindharta Do we need to update the msProtocolVersion for new serialized values like this?

Reply:
Unfortunately yes. Perhaps we can use the `fov` field to store the ortho size if it's an ortho camera ?